node/src/main/scala/com/wavesplatform/settings/WavesSettings.scala

```diff
@@ -60,4 +66,28 @@ object WavesSettings extends CustomValueReaders {
       rootConfig
     )
   }
+
+  def loadRootConfig(external: Option[File] = None): WavesSettings = {
+    val config = loadConfig(external.map(ConfigFactory.parseFile))
+    val settings = fromRootConfig(config)
+
+    // Initialize global var with actual address scheme
+    AddressScheme.current = new AddressScheme {
+      override val chainId: Byte = settings.blockchainSettings.addressSchemeCharacter.toByte
+    }
+
+    if (config.getBoolean("kamon.enable")) {
+      Kamon.addReporter(new InfluxDBReporter())
+      SystemMetrics.startCollecting()
+    }
+
+    // DO NOT LOG BEFORE THIS LINE, THIS PROPERTY IS USED IN logback.xml
```

Comment:
This comment was added for a reason. The changes you propose have broken initialization order, and now logback creates `waves.directory_IS_UNDEFINED` directory, even though it's actually defined in the config.